### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by Obote**
### What does this PR do?

- Make sure that there are no linter errors.
- Make sure that you used correct (Gitflow).
- Make sure that you documented your work.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (barackm) [inline .gitignore]: - [ ] It seems that you have forgotten to add the file `test.md` to the `.gitignore` for it to be ignored by Git, as stated in the project requirements below 👇 
image
Let us please add the test.md to the `.gitignore` 👍

File: README.md [hunk 2]
```diff
@@ -25,35 +18,15 @@ Additional description about the project and its features.
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (barackm) [inline README.md]: - [ ] Kindly let's update the getting started section with the right information, like how we can set the project locally or how to clone it, for other people to know how to do it 😉

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @Obote, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

## To Highlight
✔️  Nice PR title
✔️  Descriptive PR summary
✔️  Use of GitHub flow 👏 
✔️  Good commit messages 😉 

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

- [ ] It seems there are some issues with your interest since some linter checks did not pass yet, Kindly let us fix the issues in the CSS file to make it 😉 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Obote) [reply .gitignore]: Thanks, @barackm for the feedback to help me improve my codebase. However, I can create the test.md file but I thought the node_modules allows the file to be ignored.

Comment (barackm) [reply .gitignore]: The `node_modules/` will only let git ignore the folder `node_modules/`, so if you want to ignore something else, you will have to add it to the `.gitignore` as well. And remember we only add the `"/"` at the end if we need to ignore a folder which means you can remove the `"/"` if you need to ignore a file like the `test.md`

**Review by barackm — APPROVED** [review]
Hi @Obote, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**